istioctl/cmd/istioctl/rbac.go

```diff
@@ -84,6 +86,25 @@ istioctl experimental rbac can -s source.namespace=foo POST rating /data -a dest
 	return cmd
 }
 
+func upgrade() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "upgrade <rbac-file1> <rbac-file2> ...",
+		Short: "Upgrade Istio Authorization policy files to use the newest API",
```

Comment:
Be clear about the actions happened with this command, like it will need to talk to the API server to get some information so it doesn't work offline, and it must be used with the right k8s cluster which may not be true in a lot cases.

Considering this, probably it's better to provide 2 options:
- Automatically get the service->label mapping from the k8s cluster
- Manually provided mapping from the command line

Reply:
I will update the command usage. Let's focus on automatic fetching pod selector for now. Do you see a strong use case for the second case?